### DISCUSSION TITLE
GitHub Actions stuck in queue after API/Actions outage.

### DISCUSSION THREAD

**Discussion by 0xAndrewBlack** [community/community#30079] [Actions] [answered]
Hello everyone,
Yesterday was a huge outage as far as I know.
I have local runners setup with my Actions but It's stuck in queue-d status, and I can't cancel or delete the workflows.
Any idea or help to cancel them? (or in the worst case scenario delete the workflow)

**Comment by cron410** [comment, marked as answer]
~~I'm having the same issue. A workflow for https://github.com/packetworks/onetimesecret/ has been queued since yesterday but I'm able to run workflows in my other repos in the same organization just fine. Cancelling the run just shows a red banner at the top of the page indicating the cancel failed.~~

Edit: Triggering a workflow on a different repo in the same account after requesting to cancel the stuck workflow seems to have resolved the issue.
![Screen Shot 2022-08-19 at 1 32 48 PM](https://user-images.githubusercontent.com/3082899/185675426-2a156ebc-5e13-404d-869a-4a0a6d816cf3.png)

**Reply by 0xAndrewBlack** [reply]
After a few days I tried it again and it worked I had to do it several times but this trick did the job, thank you.

**Comment by miraclx** [comment]
Had the same issue – https://github.com/near/cargo-near/actions/runs/2919981926
